src/scripts/install-cli.ps1

```diff
@@ -14,14 +14,25 @@ $ProgressPreference="SilentlyContinue"
 
 $Feed="https://dotnetcli.blob.core.windows.net/dotnet"
 $Channel="beta"
-$DotNetFileName="dotnet-win-" + $TargetPlatform + ".1.0.0.001672.zip"
-$DotNetUrl="$Feed/$Channel/Binaries/1.0.0.001672"
+$DotNetFileName="dotnet-dev-win-" + $TargetPlatform + ".1.0.0-beta-002111.zip"
```

Comment:
2111 will apparently break builds on Windows 7: https://github.com/dotnet/corefx/pull/7376 :/
Maybe switching it up for 2131 won't be too much hassle?